### PR TITLE
DFPL-1243+1263+1270+1282+1297: Combined Migrations

### PR DESCRIPTION
**Pull request by qasimnawaz-moj**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DFPL-1243
https://tools.hmcts.net/jira/browse/DFPL-1263
https://tools.hmcts.net/jira/browse/DFPL-1282
https://tools.hmcts.net/jira/browse/DFPL-1270
https://tools.hmcts.net/jira/browse/DFPL-1297

### Change description ###

1. DFPL-1243 Migration to remove position statement from case
2. DFPL-1282 Migration to remove manageOrder temp fields
3. Remove Case Summary Document From MB22C50275
4. Migration to remove angular braces from documentNames

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java [hunk 1]
```diff
@@ -108,6 +110,15 @@ private void run1218(CaseDetails caseDetails) {
         removeSpecificPlacements(caseDetails, placementToRemove);
     }
 
+    private void run1243(CaseDetails caseDetails) {
+        var migrationId = "DFPL-1243";
+        var possibleCaseIds = List.of(1675697653441050L);
+
+        migrateCaseService.doCaseIdCheckList(caseDetails.getId(), possibleCaseIds, migrationId);
+
+        caseDetails.getData().remove(POSITION_STATEMENT_LIST_CHILD);
```
Comment (so99y) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java:119]: Are we removing the whole list of position statement child?
FYI, there is a fnuction in `MigrateCaesService.removePositionStatementChild` for removing position statement child.

**Review by so99y — COMMENTED** [review]
maybe add the following logic in `MigrateCaseService.removePositionStatementChild` to remove the collection if empty.
```
        if (positionStatementChildListResult.size() == 0) {
            return Map.of("positionStatementChildListV2", null);
        }
```

File: service/src/main/java/uk/gov/hmcts/reform/fpl/service/MigrateCaseService.java [hunk 2]
```diff
@@ -131,6 +131,10 @@ public Map<String, Object> removePositionStatementChild(CaseData caseData,
                 .filter(el -> !el.getId().equals(expectedPositionStatementId))
                 .collect(toList());
 
+        if (positionStatementChildListResult.size() == 0) {
+            return Map.of("positionStatementChildListV2", null);
```
Comment (so99y) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/service/MigrateCaseService.java:135]: should add a test case for it?

**Review by so99y — DISMISSED** [review]
LGTM

**Review by so99y — APPROVED** [review]

**Review by chak-shing-lo-justice — APPROVED** [review]

**Pull request merged**